cfndsl_ext/lambda_helper.rb

```diff
@@ -103,7 +110,11 @@ def render_lambda_functions(cfndsl, lambdas, lambda_metadata, distribution)
 
     if lambda_config.has_key?('log_retention')
       Logs_LogGroup("#{name}LogGroup") do
-        LogGroupName "/aws/lambda/#{name}"
+        if lambda_config['loggroup_name'].nil?
+          LogGroupName "/aws/lambda/#{name}"
```

Comment:
i think this should change to 
```rb
LogGroupName FnSub("/aws/lambda/${#{function_name}}")
```
as the log group name has to match the function name

Reply:
Fixed in the below commit :)

Reply:
When it will be implemented?